airgun/widgets.py

```diff
@@ -12,6 +14,88 @@
 from widgetastic_patternfly import VerticalNavigation
 
 
+class RadioGroup(Widget):
```

Comment:
Why not inheriting from `GenericLocatorWidget`? That way you wouldn't have to specify `def __init__()`